src/test/regress/expected/multi_explain.out

```diff
@@ -47,8 +47,8 @@ Sort
   ->  HashAggregate
         Group Key: remote_scan.l_quantity
         ->  Custom Scan (Citus Real-Time)
-              Task Count: 8
-              Tasks Shown: One of 8
+              Task Count: 2
```

Comment:
What is the reason for fewer task counts? Could you remind me that?

Reply:
Since we changed the shard_max_size